packages/calendar/src/index.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react'
+import {useState} from 'react'
+import getUserLocale from 'get-user-locale'
+import {useDefaults} from '@smashing/theme'
+import {CalendarProps, CalendarAppearanceType} from './types'
+import {StyledContainer, StyledInput, StyledCalendar} from './styles'
+
+const Calendar: React.FC<CalendarProps> = ({
+  onChange = () => {},
+  startValue,
+  ...props
+}) => {
+  const [open, setOpen] = useState(false)
+  const [date, setDate] = useState(startValue)
```

Comment:
if `date` is dependent on props (which results from passing it to `onChange` callback in `onDateChange` method), this means that component is controlled - which in turn indicates that local state is redundant.